workflows/scoring/validate.cwl

```diff
@@ -99,7 +100,19 @@ requirements:
                     return wiki['markdown']
                 else:
                     return None
+            
+            def check_permissions(syn, sub):
+                invalid_reasons = []
+                project_id = sub['entityId']
+
+                try:
+                    perms = syn.getPermissions(project_id, 3380061)
+                    if "READ" not in perms:
+                        invalid_reasons.append("User 'wellcomeprize' cannot read the project.")
+                except synapseclient.exceptions.SynapseHTTPError as e:
+                    invalid_reasons.append("User 'wellcomeprize' cannot read the project.")
```

Comment:
`"User 'wellcomeprize' does not have DOWNLOAD permissions on {}".format(project_id)`

add team ID for wellcome eval panel